### PR TITLE
Update saddles

### PR DESCRIPTION
**Pull request by MikkelPaulson**
## What does this do?
Updates data related to saddles.

That's it.

## How was it tested?

* Manually tested in Docker with mongosh.
* Manually ran jest against branch.

## Is there a Github issue this is resolving?
No.

## Did you update the docs in the API? Please link an associated PR if applicable.
N/A

## Here's a fun image for your troubles
external-content duckduckgo com

### CODE REVIEW

File: src/5e-SRD-Equipment.json [hunk 1]
```diff
@@ -6115,7 +6115,7 @@
     },
     "weight": 50,
     "desc": [
-      "A military saddle braces the rider, helping you keep your seat on an active mount in battle. It gives you advantage on any check you make to remain mounted. An exotic saddle is required for riding any aquatic or flying mount."
+      "An exotic saddle is required for riding any aquatic or flying mount."
```
Comment (bagelbits) [inline src/5e-SRD-Equipment.json:6118]: Should we also touch the `desc` for `Saddle, Pack`?

**Review by bagelbits — COMMENTED** [review]
One small comment but otherwise looks good!

**Comment by MikkelPaulson** [comment]
@bagelbits Seems reasonable.

**Review by bagelbits — APPROVED** [review]
I think this looks good! I'll let it sit for the rest of the day and then merge it in.

**Pull request merged**